bin/stencil-download.js

```diff
@@ -25,21 +26,24 @@ const options = {
     exclude: ['parsed', 'manifest.json', ...extraExclude],
     apiHost: cliOptions.host,
     channelId: cliOptions.channel_id,
+    overwrite: cliOptions.overwrite,
     applyTheme: true, // fix to be compatible with stencil-push.utils
     file: cliOptions.file,
 };
 
 async function run(opts) {
     const overwriteType = opts.file ? opts.file : 'files';
 
-    const answers = await inquirer.prompt([
-        {
-            message: `${'Warning'.yellow} -- overwrite local with remote ${overwriteType}?`,
-            name: 'overwrite',
-            type: 'checkbox',
-            choices: ['Yes', 'No'],
-        },
-    ]);
+    const answers = opts.overwrite
+        ? [opts.overwrite]
+        : await inquirer.prompt([
+              {
+                  message: `${'Warning'.yellow} -- overwrite local with remote ${overwriteType}?`,
```

Comment:
You can use `when` option to disable/enable question: https://github.com/SBoudrias/Inquirer.js#question

Reply:
Thanks for bringing this to my attention! I have refactored it accordingly.